substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java

```diff
@@ -158,17 +158,10 @@ public boolean consume(Queue<String> args) {
             }
             useDebugAttach = true;
             String debugAttachArg = args.poll();
-            String portSuffix = debugAttachArg.substring(debugAttach.length());
-            int debugPort = 8000;
-            if (!portSuffix.isEmpty()) {
-                try {
-                    debugPort = Integer.parseInt(portSuffix.substring(1));
-                } catch (NumberFormatException e) {
-                    NativeImage.showError("Invalid " + debugAttach + " option: " + debugAttachArg);
-                }
-            }
+            String addressSuffix = debugAttachArg.substring(debugAttach.length());
+            String address = addressSuffix.isEmpty() ? "8000" : addressSuffix.substring(1);
             /* Using agentlib to allow interoperability with other agents */
-            nativeImage.addImageBuilderJavaArgs("-agentlib:jdwp=transport=dt_socket,server=y,address=" + debugPort + ",suspend=y");
+            nativeImage.addImageBuilderJavaArgs("-agentlib:jdwp=transport=dt_socket,server=y,address=" + address + ",suspend=y");
```

Comment:
Please also update the corresponding help text in https://github.com/oracle/graal/blob/12cd99f1449b14b974a0ab1e4d0a0dd744a29293/substratevm/src/com.oracle.svm.driver/resources/HelpExtra.txt#L14